offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -234,14 +252,34 @@ class OfferToPDFConverter implements OfferExporter {
             String elementId = "product-items" + "-" + tableCount
             htmlContent.getElementById("item-table-grid").append(ItemPrintout.tableHeader(elementId))
             htmlContent.getElementById("item-table-grid")
-                    .append(ItemPrintout.tableFooter(offer.overheadRatio))
+                    .append(ItemPrintout.tableFooter(offer.overheadRatio, offer.getSelectedCustomerAffiliation()))
         } else {
             //otherwise add total pricing to table
             htmlContent.getElementById("item-table-grid")
-                    .append(ItemPrintout.tableFooter(offer.overheadRatio))
+                    .append(ItemPrintout.tableFooter(offer.overheadRatio, offer.getSelectedCustomerAffiliation()))
         }
     }
 
+    void setTaxationStatement() {
+        if(!offer.getSelectedCustomerAffiliation().country.equals(countryWithVAT)) {
+            htmlContent.getElementById("vat-cost-applicable").text("Taxation is not applied to offers outside of ${countryWithVAT}")
```

Comment:
I am wondering whether we should state that only German taxes do not apply? Otherwise, this could be interpreted as: You do not have to pay taxes for this.

Reply:
Hmm Good point I could change it to "German VAT is not applied to offers outside of Germany"?

Reply:
You could also leave it as is and we wait for feedback 🤷

Reply:
I think that makes the most sense since it's probably also important to use the correct legal terminology 👮‍♂️